dashboard-widgets.el

```diff
@@ -750,6 +756,8 @@ Argument IMAGE-PATH path to the image."
           (setq image-spec
                 (cond ((dashboard--type-is-gif-p img)
                        (create-image img))
+                      ((dashboard--type-is-webp-p img)
```

Comment:
This "cond" would change to `(dashboard--image-animated-p img) (create-image img)` for the `gif` and the `webp` cases.

Reply:
I was digging much into the images formats. Then i found out that gifs and webp can have single frame too.
So we don't need to animate those gifs and wepb with single frames.
I will try to create a function that checks if the image has multiple frame.
But if my function will return true only if image is (gif or webp) and  image has multiple frame.
I will make some changes. Tell me what do you think about it.